ajax/libs/CSS-Mint/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "CSS-Mint",
+  "filename": "css-mint.min.css",
+  "description": "Lightweight and simple to use UI Kit",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ArunMichaelDsouza/CSS-Mint.git"
+  },
+  "keywords": [
+    "UI",
+    "kit",
+    "css"
+  ],
+  "author": "Arun Michael Dsouza <amdsouza92@gmail.com>",
+  "license": "MIT",
+  "homepage": "http://arunmichaeldsouza.github.io/CSS-Mint/",
```

Comment:
Let's use `https`. 
Using https is better because it's more security for browsing to prevent the possibility the users been attacked during the redirection.
`https://arunmichaeldsouza.github.io/CSS-Mint/`